services/dynamic-sidecar/tests/unit/test_modules_notifier.py

```diff
@@ -0,0 +1,400 @@
+    SOCKET_IO_SERVICE_DISK_USAGE_EVENT,
+    SOCKET_IO_STATE_INPUT_PORTS_EVENT,
+    SOCKET_IO_STATE_OUTPUT_PORTS_EVENT,
+)
+from models_library.api_schemas_dynamic_sidecar.telemetry import (
+    DiskUsage,
+    ServiceDiskUsage,
+)
+from models_library.api_schemas_webserver.socketio import SocketIORoomStr
+from models_library.projects import ProjectID
+from models_library.projects_nodes_io import NodeID
+from models_library.services_types import ServicePortKey
+from models_library.users import UserID
+from pydantic import ByteSize, NonNegativeInt, parse_obj_as
+from pytest_mock import MockerFixture
+from pytest_simcore.helpers.monkeypatch_envs import EnvVarsDict, setenvs_from_dict
+from servicelib.utils import logged_gather
+from settings_library.rabbit import RabbitSettings
+from simcore_service_dynamic_sidecar.core.application import create_app
+from simcore_service_dynamic_sidecar.core.settings import ApplicationSettings
+from simcore_service_dynamic_sidecar.modules.notifications import (
+    PortNotifier,
+    publish_disk_usage,
+)
+from simcore_service_dynamic_sidecar.modules.system_monitor._disk_usage import (
+    DiskUsageMonitor,
+)
+from socketio import AsyncServer
+from tenacity import AsyncRetrying
+from tenacity.stop import stop_after_delay
+from tenacity.wait import wait_fixed
+
+pytest_simcore_core_services_selection = [
+    "rabbit",
+]
+
+_NUMBER_OF_CLIENTS: Final[NonNegativeInt] = 10
+
+
+@pytest.fixture
+def mock_environment(
+    monkeypatch: pytest.MonkeyPatch,
+    rabbit_service: RabbitSettings,
+    mock_environment: EnvVarsDict,
+) -> EnvVarsDict:
+    return setenvs_from_dict(
+        monkeypatch,
+        {
+            "DY_SIDECAR_SYSTEM_MONITOR_TELEMETRY_ENABLE": "true",
+            "RABBIT_HOST": rabbit_service.RABBIT_HOST,
+            "RABBIT_PASSWORD": rabbit_service.RABBIT_PASSWORD.get_secret_value(),
+            "RABBIT_PORT": f"{rabbit_service.RABBIT_PORT}",
+            "RABBIT_SECURE": f"{rabbit_service.RABBIT_SECURE}",
+            "RABBIT_USER": rabbit_service.RABBIT_USER,
+        },
+    )
+
+
+@pytest.fixture
+async def app(
+    mock_environment: EnvVarsDict,
+    mock_registry_service: AsyncMock,
+    mock_storage_check: None,
+    mock_postgres_check: None,
+    mocker: MockerFixture,
+) -> AsyncIterable[FastAPI]:
+    mocker.patch(
+        "simcore_service_dynamic_sidecar.modules.system_monitor._disk_usage._get_monitored_paths",
+        return_value=[],
+    )
+
+    app: FastAPI = create_app()
+    async with LifespanManager(app):
+        yield app
+
+
+@pytest.fixture
+async def disk_usage_monitor(app: FastAPI) -> DiskUsageMonitor:
+    return app.state.disk_usage_monitor
+
+
+@pytest.fixture
+async def socketio_server(
+    app: FastAPI,
+    socketio_server_factory: Callable[
+        [RabbitSettings], _AsyncGeneratorContextManager[AsyncServer]
+    ],
+) -> AsyncIterable[AsyncServer]:
+    # Same configuration as simcore_service_webserver/socketio/server.py
```

Comment:
So in this file you basically test that notifier thing. Please move it out of the node ports. this is indirect testing.
If we do not need that thing anywhere else (which is the case) there is no need to have it in the node ports.